.github/workflows/run_coverage.yml

```diff
@@ -48,20 +48,26 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -r requirements-dev.txt -r requirements.txt -r requirements-opt.txt
+          python -m pip install -r requirements-dev.txt -r requirements.txt
+
+      - name: Install optional dependencies
+        if: ${{ matrix.opt_req }}
+        run: python -m pip install -r requirements-opt.txt
```

Comment:
The testing matrix was set up to test and report coverage with both optional requirements installed and not installed. Testing with the optional requirements takes a lot longer because of the streaming test.